pydov/types/interpretaties.py

```diff
@@ -344,3 +344,92 @@ class LithologischeBeschrijvingen(AbstractBoringInterpretatie):
         'sourcefield': 'Y_mL72',
         'type': 'float'
     }]
+
+
+class GecodeerdeLithologieLaag(AbstractDovSubType):
+
+    _name = 'gecodeerde_lithologie_laag'
+    _rootpath = './/gecodeerdelithologie/laag'
+
+    _fields = [{
+        'name': 'diepte_laag_van',
+        'source': 'xml',
+        'sourcefield': '/van',
+        'definition': 'Diepte van de bovenkant van de laag lithologische'
+                      ' beschrijving in meter.',
```

Comment:
This needs to be "Diepte van de bovenkant van de laag gecodeerde lithologie in meter"